samples/sample_encode/src/sample_encode.cpp

```diff
@@ -1433,11 +1433,11 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
     //    return MFX_ERR_UNSUPPORTED;
     //}
 
-    if ((pParams->nMaxSliceSize) && (pParams->CodecId != MFX_CODEC_AVC))
-    {
-        PrintHelp(strInput[0], MSDK_STRING("MaxSliceSize option is supported only with H.264 encoder!"));
-        return MFX_ERR_UNSUPPORTED;
-    }
+    //if ((pParams->nMaxSliceSize) && (pParams->CodecId != MFX_CODEC_AVC))
```

Comment:
1. Please "Do not leave dead commented code" (see Coding_guidelines.md)
2. This commit intends to unblock max slice size for h265 but the fix unblock it for all codecs